optimum/commands/export/onnx.py

```diff
@@ -90,7 +90,15 @@ def parse_args_onnx(parser):
     optional_group.add_argument(
         "--trust-remote-code",
         action="store_true",
-        help="Allow to use custom code for the modeling hosted in the model repository. This option should only be set for repositories you trust and in which you have read the code, as it will execute on your local machine arbitrary code present in the model repository.",
+        help="Allows to use custom code for the modeling hosted in the model repository. This option should only be set for repositories you trust and in which you have read the code, as it will execute on your local machine arbitrary code present in the model repository.",
+    )
+    optional_group.add_argument(
+        "--no-post-process",
+        action="store_true",
+        help=(
+            "Allows to disable any post-processing done by default on the exported ONNX models. This is for example the merging of decoder"
+            " and decoder-with-past into a single ONNX with If node."
```

Comment:
```suggestion
            "Allows to disable any post-processing done by default on the exported ONNX models. For example, the merging of decoder"
            " and decoder-with-past models into a single ONNX model file to reduce memory usage."
```
I would prefer to mention what it brings than details like control flow, which need further understanding.